### ISSUE TITLE
d2l alias to framework syntax conversion 

### ISSUE THREAD

**Issue by AnirudhDagar** [d2l-ai/d2l-book#14]
`d2l.reshape(X, (1,1))` and `d2l.size(y)` need to be rendered back to pytorch/tf/mx version like `X.reshape(1,1)` and `y.size` in d2l.ai.

You can find these being rendered with the d2l alias which is not expected. You can find these rendering issues [here](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_recurrent-neural-networks/rnn-scratch.html#one-hot-encoding) and in many other sections of the book. 

Attaching a few screenshots below.

**d2l.reshape**
<img src="https://user-images.githubusercontent.com/23621655/93645510-cbe70080-fa21-11ea-978e-3a8be881a02d.png" width="50%" />

**d2l.size**
<img src="https://user-images.githubusercontent.com/23621655/93645522-d1dce180-fa21-11ea-9b59-24e067fa8b5e.png" width="50%" />

**Comment by astonzhang** [comment]
![Screen Shot 2020-10-07 at 12 33 49 PM](https://user-images.githubusercontent.com/22279212/95378950-89cf1300-0899-11eb-8a19-ee21bc159e78.png)

Similarly, in both MXNet/PyTorch, `d2l.reduce_sum(X)` is not converted back to `X.sum`

**Comment by mli** [comment]
should be fixed by https://github.com/d2l-ai/d2l-book/commit/bd4e4bee80567e6bd67494b77125ff656236775f

**Comment by AnirudhDagar** [comment]
@mli Was this issue not fixed in this [PR](https://github.com/d2l-ai/d2l-en/pull/1633)?

I can see both the difference [PR preview version](http://preview.d2l.ai/d2l-en/PR-1633/chapter_linear-networks/softmax-regression-scratch.html) and [Current Master Version](http://preview.d2l.ai/d2l-en/master/chapter_linear-networks/softmax-regression-scratch.html).

You can see that `d2l.reduce_sum` is being converted back. Similarly for section 8.5 the issue is fixed I guess.

**Comment by astonzhang** [comment]
> @mli Was this issue not fixed in this [PR](https://github.com/d2l-ai/d2l-en/pull/1633)?
> 
> I can see both the difference [PR preview version](http://preview.d2l.ai/d2l-en/PR-1633/chapter_linear-networks/softmax-regression-scratch.html) and [Current Master Version](http://preview.d2l.ai/d2l-en/master/chapter_linear-networks/softmax-regression-scratch.html).
> 
> You can see that `d2l.reduce_sum` is being converted back. Similarly for section 8.5 the issue is fixed I guess.

![Screen Shot 2021-01-14 at 1 50 30 PM](https://user-images.githubusercontent.com/22279212/104653488-89e86700-566f-11eb-9f0f-eca1a551f3e3.png)
![Screen Shot 2021-01-14 at 1 50 36 PM](https://user-images.githubusercontent.com/22279212/104653492-8b199400-566f-11eb-943c-3d39fc541ca6.png)
![Screen Shot 2021-01-14 at 1 50 22 PM](https://user-images.githubusercontent.com/22279212/104653508-910f7500-566f-11eb-9663-6bc024861065.png)

I think it's fixed.
